sio.go

```diff
@@ -115,6 +115,42 @@ type Config struct {
 	PayloadSize int
 }
 
+// EncryptedSize returns the number of bytes of an DARE 2.0
+// encrypted data stream where size is the size of the plaintext
+// in bytes. It returns an error if size is larger than 2^48.
+func EncryptedSize(size uint64) (uint64, error) {
+	if size > 1<<48 {
+		return 0, errors.New("sio: size is too large for DARE")
```

Comment:
perhaps we can use a typed error here?

Reply:
We can - will change

Reply:
No I simply mean ErrUnexpectedSize = errors.New()